LoopUI/Charts/PredictedGlucoseChart.swift

```diff
@@ -245,10 +245,46 @@ extension PredictedGlucoseChart {
     }
 
     public func setPredictedGlucoseValues(_ glucoseValues: [GlucoseValue]) {
-        predictedGlucosePoints = glucosePointsFromValues(glucoseValues)
+        let sanitizedPredictedGlucoseValues = sanitizePredictedGlucoseValues(glucoseValues)
+        predictedGlucosePoints = glucosePointsFromValues(sanitizedPredictedGlucoseValues)
     }
 
     public func setAlternatePredictedGlucoseValues(_ glucoseValues: [GlucoseValue]) {
         alternatePredictedGlucosePoints = glucosePointsFromValues(glucoseValues)
     }
 }
+
+
+// MARK: - Clipping the predicted glucose values to a suggested min and max
+import HealthKit
+
+extension PredictedGlucoseChart {
+    var suggestedPredictedGlucoseMax: HKQuantity {
```

Comment:
[nit] Is this actually "suggested"? This implies the suggestion can be overridden. Perhaps "predictedGlucoseClampMaximum" (per suggestion below).

Reply:
I think including `charted` will help indicate that this only considers the values in the glucose chart instead of all glucose values.